### ISSUE TITLE
 Some _ExternalInterface-* test files not transitioned properly for new field in ZoneHVAC:IdealLoadAirSystem

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#6320]
Issue overview
--------------
While looking for an example file that uses the new ideal loads "System Inlet Air Node Name", found this in `_ExternalInterface-actuator.idf` and ` _ExternalInterface-variable.idf`
```
  ZoneHVAC:IdealLoadsAirSystem,
    Zone3Air,                !- Name
    ,                        !- Availability Schedule Name
    NODE_3,                  !- Zone Supply Air Node Name
    ,                        !- Zone Exhaust Air Node Name
    50,                      !- System Inlet Air Node Name
    13,                      !- Maximum Heating Supply Air Temperature {C}
    0.015,                   !- Minimum Cooling Supply Air Temperature {C}
 ```
Apparently this file doesn't get run on CI.  All these field values starting with 50 need to shift down. There are three ideal loads objects like this.  Not sure if this file has any other v8.8 problems.

BTW, @rraustad it would be good to add this feature to one of the example files.   But I see we don't currently have an example file with ideal loads and a plenum zone.  Would need to make a new one, then.

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by rraustad** [comment]
I have my test file(s) that I can add to the example files.
5ZoneAirCooled_IdealLoadsAirSystems_2ReturnPlenums

**Comment by mjwitte** [comment]
@rraustad Will you be able to address at least the broken  files in the next day or so?

**Comment by rraustad** [comment]
I'll have this ready for a review tomorrow or today if I can get to it.

**Comment by rraustad** [comment]
How are the ExamplefFiles and ExampleFiles-ObjectsLink html generated? or where are they?

**Comment by Myoldmopar** [comment]
They are automatically generated.

**Comment by Myoldmopar** [comment]
https://github.com/NREL/EnergyPlus/blob/develop/doc/tools/example_file_summary.py

**Comment by mjwitte** [comment]
@tsnouidui Would you be able to test the external interface files on this branch, please?

**Comment by tsnouidui** [comment]
Sure I can do that. Would the test be to compile E+ on that branch and run the BCVTB examples with the transitioned ExternalInterface-actuator.idf, and _ExternalInterface-variable.idf located in https://github.com/NREL/EnergyPlus/tree/%236320-Some-_ExternalInterface--test-files-not-transitioned-properly-for-new-field-in-ZoneHVACIdealLoadAirSystem/testfiles?

**Comment by mjwitte** [comment]
Yes. Thanks.

**Comment by tsnouidui** [comment]
I can't for whatever reason not switch to that branch on Ubuntu (Linux). I tested it on Windows with `Git for Windows`.

When I do 

`git checkout #6320-Some-_ExternalInterface--test-files-not-transitioned-properly-for-new-field-in-ZoneHVACIdealLoadAirSystem`

and then 

`git branch` to check the branch I am on, I get that I am still on `develop`.

If I try to switch to another branch such as `update_libraries`, it works.

@Myoldmopar  @mjwitte Could you please check to see if you can reproduce?

**Comment by Myoldmopar** [comment]
Put quotes around the entire branch name.  The octothorpe # is a comment symbol in bash.

**Comment by tsnouidui** [comment]
Thanks. That works. 
Maybe it was discussed but why don't we let name of branches start with issueXXX where XXX is the number of the issue associated with the branch?

**Comment by Myoldmopar** [comment]
- Some branches aren't associated with issues.
- We give the developers creative license to name their branches in interesting ways.

:laughing:

**Comment by tsnouidui** [comment]
I see 
:laughing:

**Comment by Myoldmopar** [comment]
Addressed via #6331
